managed/src/main/java/com/yugabyte/yw/common/TemplateManager.java

```diff
@@ -97,7 +101,7 @@ public void createProvisionTemplate(
       accessKey.setKeyInfo(keyInfo);
       accessKey.save();
     } else {
-      throw new RuntimeException(Json.stringify(result));
+      throw new YWServiceException(INTERNAL_SERVER_ERROR, Json.stringify(result));
```

Comment:
There is a constructor that take JSonNode or Object as argument. That serves as a structured error message when one is available. So no need to stringify. Use that ctor when available.